client-v2/src/main/java/com/clickhouse/client/api/Client.java

```diff
@@ -385,9 +385,9 @@ public Builder addProxy(ProxyType type, String host, int port) {
             ValidationUtils.checkNonBlank(host, "host");
             ValidationUtils.checkRange(port, 1, ValidationUtils.TCP_PORT_NUMBER_MAX, "port");
 
-            this.configuration.put(String.valueOf(ClickHouseClientOption.PROXY_TYPE), type.toString());
-            this.configuration.put(String.valueOf(ClickHouseClientOption.PROXY_HOST), host);
-            this.configuration.put(String.valueOf(ClickHouseClientOption.PROXY_PORT), String.valueOf(port));
+            this.configuration.put(String.valueOf(ClickHouseClientOption.PROXY_TYPE).toLowerCase(), type.toString());
```

Comment:
`com.clickhouse.client.config.ClickHouseClientOption#getKey` should be used instead. It works now, but some config options doesn't match with enum constant name